src/views/dialogs/DisplaySettingsDialog.tsx

```diff
@@ -76,6 +78,50 @@ export const DisplaySettingsDialog = ({ setIsOpen }: ElementProps) => {
     );
   };
 
+  const colorModeOptions = () => {
+    return (
+      <Styled.ColorPreferenceRoot value={currentColorMode}>
+        {[
+          {
+            colorMode: AppColorMode.GreenUp,
+            label: STRING_KEYS.GREEN_IS_UP,
+          },
+          {
+            colorMode: AppColorMode.RedUp,
+            label: STRING_KEYS.RED_IS_UP,
+          },
+        ].map(({ colorMode, label }) => (
+          <Styled.ColorPreferenceItem
+            key={colorMode}
+            value={colorMode}
+            onClick={() => {
+              dispatch(setAppColorMode(colorMode));
+            }}
+          >
+            <Styled.ColorPreferenceLabel>
+              <Styled.DiffArrowContainer>
+                <Styled.DiffArrow
+                  direction="up"
+                  sign={
+                    colorMode === AppColorMode.GreenUp ? NumberSign.Positive : NumberSign.Negative
+                  }
+                />
+                <Styled.DiffArrow
+                  direction="down"
+                  sign={
+                    colorMode === AppColorMode.GreenUp ? NumberSign.Negative : NumberSign.Positive
+                  }
+                />
+              </Styled.DiffArrowContainer>
```

Comment:
`<DiffArrow>` is probably the wrong component for this. I think we can just use the `<Icon>` component instead